### DISCUSSION TITLE
Where to put Z endstop plug of the BL Touch with this firmware

### DISCUSSION THREAD

**Discussion by mechano** [mechano/Kingroon_KP5L_Marlin_Firmware#2] [Announcements]
With this firmware on KP5L

the Z Endstop plug has to be put on the free socket Z Max, the actual Z endstop switch has to be disconnected and can be removed from the printer.

![image](https://github.com/mechano/Kingroon_KP5L_Marlin_Firmware/assets/2874499/6f3939f8-34ac-40e7-957f-72313480fd93)

**Comment by MrDemonMinsk** [comment]
Hi. Everything is installed exactly as shown in the photo. With the latest firmware (2024), the z-axis stopped dropping on homing. I don't understand which parameter I need to correct. Everything works in the previous version, but there's no configuration file to compare.

**Comment by mechano** [comment]
Normally that problems are realted to wrong wiring.
Double check wiring.

**Comment by MrDemonMinsk** [comment]
I checked everything. Everything worked on a different firmware. I decided to update the firmware and it stopped working.

**Comment by mechano** [comment]
It's possible the other firmware uses Z endstop on Z- plug on the mainboard.
My firmware uses same position but Z+, so you have only to change the plug's position.
